cocos/dragon-bones/ArmatureCache.ts

```diff
@@ -185,32 +208,52 @@ export class AnimationCache {
 
         this.end();
     }
-
+    /**
+     * @en Is inited?
+     * @zh 检查是否已初始化。
+     */
     isInited () {
         return this._inited;
     }
-
+    /**
+     * @en Check if current state is invalid.
+     * @zh 检查当前状态是否为无效。
+     */
     isInvalid () {
         return this._invalid;
     }
-
+    /**
+     * @en Mark all cached frames as invalid.
+     * @zh 将所有缓存帧标记为无效的。
+     */
     invalidAllFrame () {
```

Comment:
invalidAllFrame -> invalidateAllFrames. Of course, can rename it in next version.